src/pages/PictureUploadBox.tsx

```diff
@@ -37,12 +39,29 @@ const PictureUploadBox = ({
   onClickDeleteImage,
   onClickSetMainImage,
   onClickAddImage,
+  onDragAddImage,
   ...props
 }: Props) => {
   const { t } = useTranslation();
 
+  const handleDrop = (e: DragEvent<HTMLElement>) => {
+    e.nativeEvent.preventDefault();
```

Comment:
Can't we just use e.preventDefault ?

https://reactjs.org/docs/events.html#overview